### ISSUE TITLE
Improve options skew errors

### ISSUE THREAD

**Issue by jakemac53** [dart-lang/build#2198]
The options skew message right now isn't necessarily very helpful, users might not realize what other process exactly is using the build daemon.

One way to solve this could be adding a required field when you connect to the daemon, which would contain a human understandable name for the client (`webdev`, `bolt`, `build_runner`, etc). Then we could log the names of the existing clients in the error so people might have a better understanding of what clients they need to shut down.

**Comment by grouma** [comment]
Yes. We can also consider adding a processId making it easy for users to kill.

**Comment by davidmorgan** [comment]
Closing as stale.
